coffee-model/coffee-model-base/src/main/java/hu/icellmobilsoft/coffee/model/base/generator/EntityIdGenerator.java

```diff
@@ -136,7 +136,7 @@ public static String generateId() {
     private static synchronized int getNextIndex() {
         generatedIndex++;
         // MAX a ZZ
-        if (generatedIndex > 1296) {
+        if (generatedIndex > 1295) {
```

Comment:
Kicsit sok már a magic number ebben az osztályban. Sok helyen szerepel a 36, ki lehetne emelni konstansba. És akkor itt is lehetne `generatedIndex > RADIX * RADIX - 1`

Reply:
Azért van szerintem ennyi magic szám, mert ezeket ha mindig számolgatni kell az sok idő (sok kicsi sokra megy).

Reply:
A konstansokat a compiler úgy is lecseréli

Reply:
kipróbáltam teszten, és bele is tettem a változót.

Reply:
Szuper, köszi